helm/superset/templates/secret-env.yaml

```diff
@@ -34,6 +34,10 @@ stringData:
     REDIS_PASSWORD: {{ .Values.supersetNode.connections.redis_password | quote }}
     {{- end }}
     REDIS_PORT: {{ .Values.supersetNode.connections.redis_port | quote }}
+    REDIS_PROTO: {{ if .Values.supersetNode.connections.use_redis_ssl }}"rediss"{{ else }}"redis"{{ end }}
+    REDIS_DB: {{ .Values.supersetNode.connections.redis_cache_db | quote }}
+    REDIS_CELERY_DB: {{ .Values.supersetNode.connections.redis_celery_db | quote }}
+    REDIS_SSL_PARAM: {{ if .Values.supersetNode.connections.use_redis_ssl }}"?ssl_cert_reqs={{ .Values.supersetNode.connections.use_redis_ssl.ssl_cert_reqs | default "CERT_NONE" }}"{{ else }}""{{ end }}
```

Comment:
nit: how about just setting a `REDIS_SSL_CERT_REQS` with `.Values.supersetNode.connections.use_redis_ssl.ssl_cert_reqs` then moving all the necessary logic to the construction of the URL

Reply:
The logic is here as we need ssl_cert_reqs only while using the ssl protocol. Are you suggesting we set the REDIS_SSL_PARAM envvar only when ssl is needed so secret-env.yaml can be something like this -   
```yaml
{{- if .Values.supersetNode.connections.use_redis_ssl }}
REDIS_SSL_CERT_REQS: {{ .Values.supersetNode.connections.use_redis_ssl.ssl_cert_reqs | default "CERT_NONE" }}
{{- end }}
```   
   
So we will now build the URL as follow - 
```python
broker_url = f"{REDIS_BASE_URL}/{env('REDIS_CELERY_DB', 0)}{'?ssl_cert_reqs='+env('REDIS_SSL_CERT_REQS') if env('REDIS_SSL_CERT_REQS') else ''}"
```

Reply:
how about something similar to:

``` yaml
# Redis Base URL
{{- if .Values.supersetNode.connections.redis_password }}
REDIS_BASE_URL=f"{env('REDIS_PROTO')}://:{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}"
{{- else }}
REDIS_BASE_URL=f"{env('REDIS_PROTO')}://{env('REDIS_HOST')}:{env('REDIS_PORT')}"
{{- end }}
{{- if .Values.supersetNode.connections.use_redis_ssl }}
REDIS_PARAMS = "?ssl_cert_req={env('REDIS_SSL_CERT_REQS'}"
{{ - else }}
REDIS_PARAMS = ""
{{- end}}
CACHE_REDIS_URL = f"{REDIS_BASE_URL}/{env('REDIS_DB', 1)}{REDIS_PARAMS}"
CELERY_REDIS_URL = f"{REDIS_BASE_URL}/{env('REDIS_DB', 0)}{REDIS_PARAMS}"
```

Reply:
Ahh, gotcha. Will change it

Reply:
Done the changes, can you please review?